components/script/dom/request.rs

```diff
@@ -666,9 +666,9 @@ impl BodyMixin for Request {
     }
 }
 
-impl Into<NetTraitsRequestCache> for RequestCache {
-    fn into(self) -> NetTraitsRequestCache {
-        match self {
+impl From<RequestCache> for NetTraitsRequestCache {
+    fn from(val: RequestCache) -> Self {
+        match val {
```

Comment:
```suggestion
impl From<RequestCache> for NetTraitsRequestCache {
    fn from(value: RequestCache) -> Self {
        match value {
```

Reply:
I've gone ahead and made these changes.

Reply:
Apologies, I was preoccupied with some personal work and couldn't attend to the changes requested. 
Thank you for making the commit.

Reply:
No worries. Thank you for the contribution!